sdk/storage/Azure.Storage.Files.Shares/tests/FileClientTests.cs

```diff
@@ -84,40 +84,40 @@ public void FilePathsParsing()
             Uri uri1 = new Uri("http://dummyaccount.file.core.windows.net/share/dir1/dir2/file.txt");
             var builder1 = new ShareUriBuilder(uri1);
             var fileClient1 = new ShareFileClient(uri1);
-            TestHelper.AssertCacheableProperty("file.txt", () => fileClient1.Name);
-            TestHelper.AssertCacheableProperty("dir1/dir2/file.txt", () => fileClient1.Path);
+            Assert.AreEqual("file.txt", fileClient1.Name);
```

Comment:
It weirds me out a little bit that I needed to change this.  Same with the other instances of `TestHelper.AssertCacheableProperty(...)` in `FileClientTests`.